src/user-guide/running-workflows/index.rst

```diff
@@ -1,7 +1,22 @@
+.. _RunningWorkflows:
+
 Running Workflows
 =================
 
+.. cylc-scope::
+
```

Comment:
Unexpected.

When the scope is changed it should be reset afterwards e.g:

```
.. cylc-scope:: flow.cylc[scheduling]

Use the :cylc:conf`initial cycle point`.

.. cylc-scope::
```

If this is needed here it is possible that the scope was not reset in another section?

Reply:
I'll check